### PR TITLE
RL Worker and required util to interact with API

### PR DESCRIPTION
**Pull request by vkartik97**
The PR consist of:
1. `rl_submission_worker`: Script to run Docker evaluations
2. `worker_util`: Module for remote evaluators to interact with API.

### CODE REVIEW

File: scripts/workers/rl_submission_worker.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        signal.signal(signal.SIGINT, self.exit_gracefully)
+        signal.signal(signal.SIGTERM, self.exit_gracefully)
+
+    def exit_gracefully(self, signum, frame):
+        self.kill_now = True
+
+
+logger = logging.getLogger(__name__)
+
+AUTH_TOKEN = os.environ.get("AUTH_TOKEN", "x")
```
Comment (rishabhbhardwaj) [inline scripts/workers/rl_submission_worker.py]: @fristonio @vkartik97 is there a specific reason to have these as envs. Can we instead have these as arguments to the main? These can be passed while starting the container. Let me know your thoughts.

File: scripts/workers/rl_submission_worker.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        QUEUE_NAME=QUEUE_NAME,
+    )
+    print("String RL Worker for {}".format(api.get_challenge_by_queue_name()["title"]))
+    killer = GracefulKiller()
+    while True:
+        logger.info(
+            "Fetching new messages from the queue {}".format(QUEUE_NAME)
+        )
+        message = api.get_message_from_sqs_queue()
+        print(message)
```
Comment (rishabhbhardwaj) [inline scripts/workers/rl_submission_worker.py]: Use logger

File: scripts/workers/rl_submission_worker.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+            submission = api.get_submission_by_pk(submission_pk)
+            if submission:
+                if submission.get("status") == "finished":
+                    message_receipt_handle = message.get("receipt_handle")
+                    api.delete_message_from_sqs_queue(message_receipt_handle)
+                elif submission.get("status") == "running":
+                    continue
+                else:
+                    message_receipt_handle = message.get("receipt_handle")
+                    print(
```
Comment (rishabhbhardwaj) [inline scripts/workers/rl_submission_worker.py]: Use logger in place of prints.

File: scripts/workers/rl_submission_worker.py [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+                        "Processing message body: {}".format(message_body)
+                    )
+                    process_submission_callback(message_body, api)
+                    api.delete_message_from_sqs_queue(message.get("receipt_handle"))
+        time.sleep(5)
```
Comment (rishabhbhardwaj) [inline scripts/workers/rl_submission_worker.py]: Can we read this from config or make a constant variable for this?

**Comment by vkartik97** [comment]
@rishabhbhardwaj please review

**Pull request merged**